### PR TITLE
New utility functions are added in utils.py and tests are added in te…

### PR DESCRIPTION
**Pull request by ibrahimAnis**
# Overview
Referencing #179 issue in this PR request.

New functions are added in utils.py file.These functions are as follows:-

- vertices_from_actor:- This function will retrieve the vertices and return it in the form of numpy array.

- compute_bounds:- This function will compute the bounds using the reference of actor.
 
- modify_actor:- This function will call the Modified function using the reference of actor.

Test for vertices_from_actor function is added only. I could not be able to write test for compute_bounds and modify_actor function

### CODE REVIEW

File: fury/utils.py [hunk 1]
```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
     norm[faces[:, 2]] += n
     normalize_v3(norm)
     return norm
+
+def vertices_from_actor(actor):
+
+    """ This function will return vertices from actor
+    """
+    vtkPoint = actor.GetMapper().GetInput()
+    points = vtkPoint.GetPoints()
```
Comment (skoudoro) [inline fury/utils.py]: Can you do it in one line: ` points =  actor.GetMapper().GetInput().GetPoints()`

File: fury/utils.py [hunk 2]
```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
...
+    data = points.GetData()
```
Comment (skoudoro) [inline fury/utils.py]: Can you do it in one line: 

```python
data = numpy_support.vtk_to_numpy(points.GetData())
return data
```

File: fury/utils.py [hunk 3]
```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
...
+    return numpy_support.vtk_to_numpy(data)
+
+def compute_bounds(actor):
+
+    """ This function will compute the bounds
+    """
```
Comment (skoudoro) [inline fury/utils.py]: same comments as above:
- No empty line
- start docstring with a verb
- finish with a period
- document input parameters and return

File: fury/utils.py [hunk 4]
```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
...
+    vtkPoint = actor.GetMapper().GetInput()
+    vtkPoint.ComputeBounds()
+
+def modify_actor(actor):
+
+    """This function will modify
+    """
```
Comment (skoudoro) [inline fury/utils.py]: same comments as above:
- No empty line
- start docstring with a verb
- finish with a period
- document input parameters and return

File: fury/utils.py [hunk 5]
```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
...
+    return numpy_support.vtk_to_numpy(data)
+
+def compute_bounds(actor):
+
+    """ This function will compute the bounds
+    """
+    vtkPoint = actor.GetMapper().GetInput()
+    vtkPoint.ComputeBounds()
+
+def modify_actor(actor):
```
Comment (skoudoro) [inline fury/utils.py]: Can you rename it `update_actor`?

File: fury/utils.py [hunk 6]
```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
...
+
+    """This function will modify
+    """
+    vtkPoint = actor.GetMapper().GetInput()
+    points = vtkPoint.GetPoints()
```
Comment (skoudoro) [inline fury/utils.py]: same comment as above:  `points = actor.GetMapper().GetInput().GetPoints()`

File: fury/tests/test_utils.py [hunk 7]
```diff
@@ -254,6 +257,65 @@ def test_rotate(interactive=False):
         red_sum_new = arr[..., 0].sum()
         npt.assert_equal(red_sum_new > red_sum, True)
 
+def test_vertices_from_actor():
+
```
Comment (skoudoro) [inline fury/tests/test_utils.py]: No need of this empty line

File: fury/tests/test_utils.py [hunk 8]
```diff
@@ -254,6 +257,65 @@ def test_rotate(interactive=False):
...
+    res = fp.repeat_primitive(verts, faces, centers=centers, colors=colors,
+                              scale=scale)
+
+    big_verts = res[0]
+    big_faces = res[1]
+    big_colors = res[2]
+    actor = get_actor_from_primitive(big_verts, big_faces, big_colors)
+    actor.GetProperty().BackfaceCullingOff()
+    res_vertices = vertices_from_actor(actor)
+    npt.assert_array_almost_equal(my_vertices, res_vertices)
```
Comment (skoudoro) [inline fury/tests/test_utils.py]: Nice test! good job!

**Comment by skoudoro** [comment]
do not forget:
- to fix the pep8 too
- to add a test for `update_actor`

**Comment by ibrahimAnis** [comment]
I have made a new PR #213. The Pull Request have updated test files. Please review it and give me feedback

**Pull request closed without merging**